presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/common/FlippableBox.kt

```diff
@@ -82,6 +82,7 @@ fun FlippableBox(
             }
             .graphicsLayer {
                 rotationY = nativeAnimatedRotationAngle + if (isFront) 0f else 180f
+                cameraDistance = 18f * density
```

Comment:
```suggestion
                cameraDistance = 18.dp.toPx()
```
density 를 곱해서 dp 을 직접 계산하는 것도 좋지만, 이게 density 를 잘 모르는 사람한테는 dp 구하는 계산식인지 모르기도 해서..!
18.dp.toPx() 으로 작성해서 동일한 표현이지만 조금 더 친숙해보이게 작성해보는 것은 어떨까?!

Reply:
아아아 좋은거 같아..! 가시성 있게 바꿀겡~~~~~ 나이스~